api/init.go

```diff
@@ -14,13 +14,11 @@ func InitDefault() error {
 }
 
 // Init loads Sentinel general configuration from the given YAML file
-// and initializes Sentinel. Note that the logging module will be initialized
-// using the configuration from system environment or the default value.
+// and initializes Sentinel.
 func Init(configPath string) error {
```

Comment:
Is panic necessary if an error occurs when the sentinel is initializing?

Reply:
Maybe we could just return the error and let user decide how to handle it.

Reply:
It's ok.